helm/values.yaml

```diff
@@ -64,9 +64,16 @@ deploymentSelectorMatchLabels:
 ###########
 traceEnricherConfig:
   name: hypertrace-trace-enricher-config
-  flink:
-    sink:
-      topic: enriched-structured-traces
+  kafka:
+    streams:
+      config:
+        metricsRecordingLevel: INFO
+        numStreamThreads: 2
+        bootstrapServers: "bootstrap:9092"
+  schema:
```

Comment:
Need to fix accordingly.

Reply:
Addressed, there is not override for this.